_test_server/lib/src/handlers/test/http/reflect_handler.dart

```diff
@@ -45,7 +45,7 @@ class ReflectHandler extends Handler {
       final contentType = MediaType(
           request.headers.contentType!.primaryType,
           request.headers.contentType!.subType,
-          request.headers.contentType!.parameters as Map<String, String>?);
+          request.headers.contentType!.parameters.map((key, val) => MapEntry(key, val ?? '')));
```

Comment:
`contentType.parameters` is of the type `Map<String, String?>`, this cannot be directly casted to `Map<String, String>?`